app/views/documents/show/_withdrawn_notice.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render "govuk_publishing_components/components/notice", {
+  title: I18n.t("documents.show.withdrawn.title",
+                document_type: @document.document_type.label.downcase,
+                withdrawn_date: @edition.status.created_at.strftime("%d %B %Y"))
+} do %>
+  <%= @edition.status.details.public_explanation %>
+  <p>
+  <p>
+  <%= link_to "Change public explanation", withdraw_path(@document), class: "govuk-link govuk-link--no-visited-state" %>
```

Comment:
Would we be able to move this outside of the govspeak block? the govspeak overrides the `govuk-link--no-visited-state`

see:

![screen shot 2019-01-30 at 15 10 58](https://user-images.githubusercontent.com/282717/51990627-596a8b00-24a1-11e9-8611-1cf1a6be5c5b.png)

Reply:
As discussed @alex-ju is planning to raise a card about this. Issue seems to be notice component only expects govspeak and not HTML: https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_notice.html.erb